packages/db/src/migrations/20240516171520_ambassador_upload.ts

```diff
@@ -0,0 +1,31 @@
+import { type Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('onboarding_sessions')
+    .addColumn('uploaded_by_id', 'text')
+    .execute();
+
+  await db.schema
+    .alterTable('onboarding_sessions')
+    .addForeignKeyConstraint(
+      'fk_uploaded_by_id_admins_id',
+      ['uploaded_by_id'],
+      'admins',
+      ['id']
+    )
+    .onDelete('set null')
```

Comment:
If the admin that uploaded the Onboarding Session is removed from the `admins` table, the column `Uploaded By` will change to null. There would have to be another db saving the name of the uploaders if we wanted to prevent this

Reply:
Love it! Can we actually combine this statements into one?

```ts
await db.schema
  .alterTable('onboarding_sessions')
  .addColumn('uploaded_by_id', 'text', (column) => {
    return column.references('admins.id').onDelete('set null');
  })
  .execute();
```